nucleus/admin/config-api/src/test/java/org/glassfish/config/support/TranslatedValueTest.java

```diff
@@ -85,4 +85,10 @@ public void envTranslationRequiredWithDefault() {
         assertEquals("defaultVariable", TranslatedConfigView.expandConfigValue("${ENV=NOT_EXISTING_VARIABLE:defaultVariable}"));
         assertEquals("default:variable", TranslatedConfigView.expandConfigValue("${ENV=NOT_EXISTING_VARIABLE:default:variable}"));
     }
+
+    @Test
+    public void envTranslationRequiredWithDefaultMultiple(){
+        System.out.println("envTranslationRequired");
```

Comment:
That's not the name of the test.
```suggestion
        System.out.println("envTranslationRequiredWithDefaultMultiple");
```